bindings/matlab/+iDynTreeWrappers/generalizedGravityForces.m

```diff
@@ -0,0 +1,36 @@
+function g = generalizedGravityForces(KinDynModel)
+
+    % GENERALIZEDGRAVITYFORCES retrieves the generalized gravity forces 
+    %                               given the reduced model.
+    %
+    % This matlab function wraps a functionality of the iDyntree library.                     
+    % For further info see also: http://wiki.icub.org/codyco/dox/html/idyntree/html/
```

Comment:
That link is quite old, and actually the current documentation system is broken and we should change it soon. I would simply use https://github.com/robotology/idyntree as the entry point for documentation.
```suggestion
    % For further info see also: https://github.com/robotology/idyntree
```